### PR TITLE
feat: Connect history page to server

### LINKED ISSUE #14: feat: Connect history page to server
For temperature and humidity charts.

### PR DESCRIPTION
**Pull request by jonlld**

### CODE REVIEW

File: src/components/organisms/History.tsx [hunk 1]
```diff
@@ -71,39 +71,55 @@ export default function History() {
...
-  const labels = [
-    "Monday",
-    "Tuesday",
-    "Wednesday",
-    "Thursday",
-    "Friday",
-    "Saturday",
-    "Sunday",
-  ];
+  const labels = dataLabels;
```
Comment (Takashicc) [inline src/components/organisms/History.tsx:100]: Do you have to copy this variable?

File: src/components/organisms/History.tsx [hunk 2]
```diff
@@ -71,39 +71,55 @@ export default function History() {
...
 
+    // TODO
     const res = await fetch(
       `${url}/device-data/${device_id}/historical/week`,
       options
     );
     const data = await res.json();
-
     console.log("data: ", data);
+
```
Comment (Takashicc) [inline src/components/organisms/History.tsx]: Change `TODO` to `TODO Implement catch`.
Replace `fetch` to `axios`.

File: src/components/organisms/History.tsx [hunk 3]
```diff
@@ -169,3 +181,10 @@ export default function History() {
     </>
   );
 }
+
+// data example - array of objects
+// date: "2022/09/18";
+// max_humid: 72;
+// max_temp: 31.7;
+// min_humid: 60;
+// min_temp: 25.6;
```
Comment (Takashicc) [inline src/components/organisms/History.tsx]: Delete this comments.

**Review by Takashicc — CHANGES_REQUESTED** [review]

File: src/components/organisms/History.tsx (hunk 2 above)
Comment (Takashicc) [reply src/components/organisms/History.tsx]:
```ts
const res = await axios.get(${url}/device-data/${device_id}/historical/week`)
const data = res.data;
```

Also delete the `options` too.

Comment (jonlld) [reply src/components/organisms/History.tsx]: Done

File: src/components/organisms/History.tsx (hunk 1 above)
Comment (jonlld) [reply src/components/organisms/History.tsx:100]: maybe can use state directly but it needs to be named 'labels' to work in the chart and have kept state variable naming consistent

File: src/components/organisms/History.tsx (hunk 3 above)
Comment (jonlld) [reply src/components/organisms/History.tsx]: ok

**Review by Takashicc — APPROVED** [review]

**Pull request merged**